9c-internal/chart/templates/service.yaml

```diff
@@ -9,6 +9,9 @@ metadata:
     service.beta.kubernetes.io/aws-load-balancer-scheme: internet-facing
     service.beta.kubernetes.io/aws-load-balancer-type: "external"
```

Comment:
```service.beta.kubernetes.io/aws-load-balancer-type: "external"```
Is this part still required?

Reply:
just existing options, i'm not sure those annotations still work.